images/macos/tests/Python.Tests.ps1

```diff
@@ -8,8 +8,8 @@ Describe "Python3" -Skip:($os.IsVenturaArm64) {
         "python3 --version" | Should -ReturnZeroExitCode
     }
 
-    It "Python 3 is installed under /usr/local/bin" {
-        Get-WhichTool "python3" | Should -BeLike "/usr/local/bin*"
+    It "Python 3 is installed under /usr/bin" {

```

Comment:
we test exactly `/usr/local` here

Reply:
thus, this should be addressed in other way.
"python3" lives in /usr/bin 

if that's unexpected, ok, let's fix that